analysis/civic/examples/transform/civic:aid6.json

```diff
@@ -0,0 +1,237 @@
+{
+    "statements": [
+        {
+            "id": "civic:aid6",
+            "description": "L858R is among the most common sensitizing EGFR mutations in NSCLC, and is assessed via DNA mutational analysis, including Sanger sequencing and next generation sequencing methods. Tyrosine kinase inhibitor afatinib is FDA approved, and is recommended (category 1) by NCCN guidelines along with erlotinib, gefitinib and osimertinib as first line systemic therapy in NSCLC with sensitizing EGFR mutation.",
+            "direction": "supports",
+            "evidence_level": "civic.amp_level:tier_i_-_level_a",
+            "proposition": "proposition:124",
+            "variation_descriptor": "civic:vid33",
+            "therapy_descriptor": "civic:tid146",
+            "disease_descriptor": "civic:did8",
+            "method": "method:002",
+            "support_evidence": [
+                "support_evidence:493",
+                "support_evidence:494",
+                "support_evidence:495",
+                "support_evidence:496",
+                "support_evidence:497",
+                "support_evidence:498",
+                "support_evidence:499"
+            ],
```

Comment:
Changed `support_evidence` to `supported_by`. For CIViC EIDs, I used `civic:eid###` rather than `civic:eid.###` to stay consistent with what we have been using. If you'd rather `civic:eid.###`, I can make that change.